http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -16,39 +15,39 @@ sealed trait LeoPubsubMessage {
 }
 
 object LeoPubsubMessage {
-  final case class StopUpdate(updatedMachineConfig: RuntimeConfig, clusterId: Long, traceId: Option[TraceId])
+  final case class StopUpdateMessage(updatedMachineConfig: RuntimeConfig, runtimeId: Long, traceId: Option[TraceId])
```

Comment:
Hm, I changed it because it wasn't clear to me always that this was a pubsub message since we often `import org.broadinstitute.dsde.workbench.leonardo.monitor.LeoPubsubMessage._`